content/en/logs/log_collection/ios.md

```diff
@@ -85,7 +84,7 @@ import DatadogLogs
 Datadog.initialize(
 	with: Datadog.Configuration(
 		clientToken: "<client token>",
-		env: "<environment>",
```

Comment:
This doesn't work inside the tabs, it looks like this:
```
env: "<environment>",<span class="js-region-param region-param" data-region-param="ios_site_swift"></span>
```
You can break these out into separate sections like you did in content/en/logs/log_collection/android.md